django/db/migrations/autodetector.py

```diff
@@ -576,6 +591,134 @@ def generate_renamed_models(self):
+
+                        dependencies = list(dependencies) + [
+                            (rem_app_label, model_name, None, "alter")
+                        ]
+                        model_state.options["managed"] = False
+                        self.add_operation(
+                            app_label,
+                            operations.CreateModel(
+                                name=model_state.name,
+                                fields=[d for d in model_state.fields.items()],
+                                options=model_state.options,
+                                bases=model_state.bases,
+                                managers=model_state.managers,
+                            ),
+                            dependencies=dependencies,
+                            beginning=True,
+                        )
+                        model = self.to_state.apps.get_model(app_label, model_name)
+                        fields = model._meta.get_fields(include_hidden=False)
+                        dependencies = list(dependencies) + [
+                            (app_label, model_name, None, True),
+                        ]
+                        alter_field_deps = []
+
+                        for field in fields:
+                            if (
+                                field.is_relation
+                                and field.remote_field.related_model != model
+                            ):
+                                self.add_operation(
+                                    field.related_model._meta.model_name,
+                                    operations.AlterField(
+                                        model_name=field.related_model._meta.model_name,
+                                        name=field.remote_field.name,
+                                        field=field.remote_field,
+                                    ),
+                                    dependencies=dependencies,
+                                )
+                                self.already_alter_fields.add(
+                                    (
+                                        field.related_model._meta.model_name,
+                                        field.related_model._meta.model_name,
+                                        field.remote_field.name,
+                                    )
+                                )
+                                alter_field_deps.append(
+                                    (
+                                        rem_app_label,
+                                        field.related_model._meta.model_name,
+                                        field.remote_field.name,
+                                        "alter",
+                                    ),
+                                )
+                        dependencies = [
+                            op for op in chain(dependencies, alter_field_deps)
+                        ]
```

Comment:
Again, `dependencies` is a list, `alter_field_deps` is a list -- or am I missing something?
```suggestion
                        dependencies += alter_field_deps
```